src/Components/Crafting/CraftingWindow.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import GeneralWindow from 'Components/GeneralComponents/GeneralWindow';
+
+export default function CraftingWindow({ unmountCraftingWindow }) {
+	return (
+		<GeneralWindow unmountGeneralWindow={unmountCraftingWindow} rightSided={true}>
```

Comment:
please change `rightSided={true}` to `rightSided`

Reply:
Done.